backend/src/property_handlers.py

```diff
@@ -54,7 +54,7 @@ def post_property():
                         schema: ErrorResponse
     """
     try:
-        payload = AddPropertyRequest().load(request.get_json())
+        payload = AddPropertyRequest().load(request.get_json(), transient=True)
```

Comment:
🤔 confused here. Looking at the [api reference](https://marshmallow-sqlalchemy.readthedocs.io/en/latest/api_reference.html) transient disassociates transactions from a session. Why are we doing this? If this is necessary, why use `session.add()` / `session.commit()` below?

Reply:
We're loading a JSON payload from an API request into an object that we are then going to write to the DB. At the time it is loaded there is no DB session in scope and so the object is transient. It becomes associated with the DB session when we .add() it.

Reply:
gotcha, sounds good